lib/tasks/lockout_moves.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :lockout_moves do
+  desc 'lockouts a move which containts a MoveLockut in its generic events'
+  task lockout_moves: :environment do
+    Move.all.each do |move|
+      move_events = move.generic_events
+
+      move_events.each do |event|
+        if event.type == 'GenericEvent::MoveLockout'
+          move.is_lockout = true
+          break
+        end
+      end
+    end
```

Comment:
```suggestion
    move_ids = GenericEvent::MoveLockout.pluck(:eventable_id)
    Move.where(id: move_ids).update_all(is_lockout: true)
```

I think this will be a quicker way of running the query - I'm concerned that going through every move and then every event will be quite slow.